### PR TITLE
 INFRA: Initialize Unit Test 

### PR DESCRIPTION
**Pull request by Pwongchaiya**
Closed: #7

### CODE REVIEW

File: InternTrack.Portal.Web.Tests.Unit/InternTrack.Portal.Web.Tests.Unit.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.1" />
+    <PackageReference Include="Tynamix.ObjectFiller" Version="1.5.8" />
+    <PackageReference Include="xunit" Version="2.5.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
```
Comment (glhays) [inline InternTrack.Portal.Web.Tests.Unit/InternTrack.Portal.Web.Tests.Unit.csproj:27]: Remove section not needed

File: InternTrack.Portal.Web.Tests.Unit/Services/DeleteMe.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// -------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace InternTrack.Portal.Web.Tests.Unit.Services
```
Comment (glhays) [inline InternTrack.Portal.Web.Tests.Unit/Services/DeleteMe.cs:13]: This should be under Foundations

Comment (Pwongchaiya) [reply InternTrack.Portal.Web.Tests.Unit/Services/DeleteMe.cs:13]: @glhays Do you mean the Service folder or the DeleteMe.cs either way that's how he had it set up in the video

Comment (glhays) [reply InternTrack.Portal.Web.Tests.Unit/Services/DeleteMe.cs:13]: @Pwongchaiya Ok then... just mark as resolved, will be deleted soon when a proper test is initiated.

File: InternTrack.Portal.Web.Tests.Unit/InternTrack.Portal.Web.Tests.Unit.csproj (hunk 1 above)
Comment (Pwongchaiya) [reply InternTrack.Portal.Web.Tests.Unit/InternTrack.Portal.Web.Tests.Unit.csproj:27]: @glhays Are you asking me to delete lines 27-29?

Comment (glhays) [reply InternTrack.Portal.Web.Tests.Unit/InternTrack.Portal.Web.Tests.Unit.csproj]: @Pwongchaiya Yes that group is an artifact of vs, does nothing and it just cleans up the file by removing.

**Review by glhays — APPROVED** [review]

**Pull request merged**